src/controllers/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './category.controller';
 export * from './good-entry.controller';
 export * from './oracle.controller';
 export * from './ping.controller';
+export * from './good-type.controller';
```

Comment:
You should rename the controller file as type.controller.ts

```suggestion
export * from './type.controller';
```